modules/vaos/config/routes.rb

```diff
@@ -43,7 +43,9 @@
   end
 
   namespace :v2, defaults: { format: :json } do
+    get '/appointments', to: 'appointments#index'
     get '/locations/:location_id/clinics', to: 'clinics#index'
     get '/patients/', to: 'patients#index'
+    end
```

Comment:
Extra `end` here not needed

Reply:
ack, didn't rerun my tests locally after merging.